plugins/communications-app/BodyForm/index.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Form } from '@edx/paragon';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import TextEditor from '@communications-app/src/components/bulk-email-tool/text-editor/TextEditor';
+import { useSelector, useDispatch } from '@communications-app/src/components/bulk-email-tool/bulk-email-form/BuildEmailFormExtensible/context';
+import { actionCreators as formActions } from '@communications-app/src/components/bulk-email-tool/bulk-email-form/BuildEmailFormExtensible/context/reducer';
+
+import messages from './messages';
+
+const BodyForm = () => {
+  const intl = useIntl();
+  const formData = useSelector((state) => state.form);
+  const dispatch = useDispatch();
+  const { body, isFormSubmitted = false } = formData;
```

Comment:
Why don't do it in this way?

```suggestion
  const dispatch = useDispatch();
  const { body, isFormSubmitted = false } = useSelector(({ form }) => form);
```

Or do you consider is more clear how is it right now, I would like read your thoughts.

Reply:
I think this way is easier to read due to if you read it once and you know quickly that is a property of the state